CHANGES.md

```diff
@@ -8,18 +8,195 @@ To be released.
+    constructor.  [[#1164], [#1492]]
+ -  Added `BlockHeader.HashAlgorithm` property.  [[#1492]]
+ -  Added `BlockHeader(PreEvaluationBlockHeader, HashDigest<SHA256>)`
+    overloaded constructor.  [[#1164], [#1492]]
+ -  Added `BlockHeader(PreEvaluationBlockHeader, HashDigest<SHA256>, BlockHash)`
+    overloaded constructor.  [[#1164], [#1492]]
+ -  Added `IBlockMetadata` interface.  [[#1164], [#1492]]
+     -  `Block<T>` became to implement `IBlockMetadata` interface.
+     -  `BlockHeader` became to implement `IBlockMetadata` interface.
+     -  `BlockMetadata` became to implement `IBlockMetadata` interface.
+     -  `BlockContent<T>` became to implement `IBlockMetadata` interface.
+     -  `PreEvaluationBlockHeader` became to implement `IBlockMetadata`
+        interface.
+     -  `PreEvaluationBlock<T>` became to implement `IBlockMetadata` interface.
+     -  `BlockDigest` became to implement `IBlockMetadata` interface.
+ -  Added `IBlockContent<T>` interface.  [[#1164], [#1492]]
+     -  `Block<T>` became to implement `IBlockContent<T>` interface.
+     -  `BlockContent<T>` became to implement `IBlockContent<T>` interface.
+     -  `PreEvaluationBlock<T>` became to implement `IBlockContent<T>`
+        interface.
+ -  Added `IPreEvaluationBlockHeader` interface.  [[#1164], [#1492]]
+     -  `Block<T>` became to implement `IPreEvaluationBlockHeader` interface.
+     -  `BlockHeader` became to implement `IPreEvaluationBlockHeader` interface.
+     -  `PreEvaluationBlockHeader` became to implement
+        `IPreEvaluationBlockHeader` interface.
+     -  `PreEvaluationBlock<T>` became to implement `IPreEvaluationBlockHeader`
+        interface.
+ -  Added `IPreEvaluationBlock<T>` interface.  [[#1164], [#1492]]
+     -  `Block<T>` became to implement `IPreEvaluationBlock<T>` interface.
+     -  `PreEvaluationBlock<T>` became to implement `IPreEvaluationBlock<T>`
+        interface.
+     -  `ActionEvaluator<T>.Evaluate()` method's `block` parameter became to
+        take `IPreEvaluationBlock<T>` (was `IBlock<T>`).
+ -  Added `IBlockHeader` interface.  [[#1146], [#1164], [#1492]]
+     -  `Block<T>` became to implement `IBlockHeader` interface.
+     -  `BlockHeader` became to implement `BlockHeader` interface.
+ -  Added `BlockMetadataExtensions` static class.  [[#1164], [#1492]]
+ -  Added `BlockContentExtensions` static class.  [[#1164], [#1492]]
+ -  Added `BlockMarshaler` static class.  [[#1164], [#1492]]
+ -  Added `BlockDigest.Index` property.  [[#1492]]
+ -  Added `BlockDigest.Hash` property.  [[#1492]]
+ -  Added `BlockDigest.StateRootHash` property.  [[#1492]]
+ -  Added `BlockDigest.GetHeader()` method.  [[#1492]]
+ -  Added `StateStoreExtensions` static class.  [[#1128], [#1146], [#1492]]
+ -  Added `StoreExtensions.GetStateRootHash()` extension method.
+    [[#1128], [#1146], [#1492]]
+ -  Added `DelayedRenderer<T>.HashAlgorithmGetter` property.  [[#1492]]
+ -  `BlockDigest` became to implements `IBlockExcerpt`.
```

Comment:
```suggestion
 -  `BlockDigest` became to implement `IBlockExcerpt`.
```

Reply:
I'm going to fix the typo in the PR #1507.

Reply:
Fixed in bcdd46e464acba2a4eaaceccf753b2e9390e0d5e.